manifests/charts/istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -4,6 +4,12 @@
     # Refer to https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#21-trust-domain
     trustDomain: {{ .Values.global.trustDomain | default "cluster.local" | quote }}
 
+    # The namespace to treat as the administrative root namespace for Istio configuration.
+    # When processing a leaf namespace Istio will search for declarations in that namespace first
```

Comment:
Sorry  - what is leaf namespace here?

Reply:
emm, it's old description from the API definition:
https://github.com/istio/api/blob/57c66a1898718c125adc8f5f738d8083a53bd647/mesh/v1alpha1/config.proto#L252-L260
If I understand correctly, the leaf namespace specifies the namespace here end-user deploys the application and corresponding istio configurations. The search for istio configurations should fall back to `rootNamespace` if nothing find in the leaf namespace.

Reply:
Thanks, that makes sense, yep, it is the istio resources that are deployed to a given namespace (vs global in the root namespace.)